SConstruct

```diff
@@ -502,9 +502,10 @@ def add_cluster_analysis(w):
         where N is the ranking of vlads untrimmed deduplication, and M is the multiplicity of said deduplication."""
         # This option controls which sequences get joined on in the merge for the partis_seqmeta file, which has
         # orig/new names, joined on sequence from the other file
-        sources = {'--partis-seqmeta': c['partis_seqmeta'],
+	sources = {'--partis-seqmeta': c['partis_seqmeta'],
```

Comment:
Are you sure this indentation is right?

Reply:
Some of the indentation looks weird in git to me even though in vim it looks the same as the preceding lines.. I can try to fix

Reply:
Check to see if there's a tab in there maybe?

Reply:
Done